apps/redi-connect/src/components/organisms/EditableMentoringTopics.tsx

```diff
@@ -44,56 +46,114 @@ const categoriesByGroup = groupBy(CATEGORIES, (category) => category.group)
 
 const formCategoryGroups = objectEntries(CATEGORY_GROUPS)
```

Comment:
Do you mean using `objectEntries(groups)` instead of ` Object.fromEntries(...)`?  Im not sure, that would work here, because we need an object and `objectEntries` returns an array. Let me know if I misunderstood the comment :)